Library/Formula/collectd.rb

```diff
@@ -80,4 +81,14 @@ def plist; <<-EOS.undent
     </plist>
     EOS
   end
+
+  test do
+    begin
+      system sbin/"collectd"
+      sleep 2
+      assert File.exist?("#{var}/run/collectd.pid")
```

Comment:
This test will fail under sandbox. As we don't allow to write files outside of temporary directory during the test. Can you set this file to builldpath by argument or something?